assets/js/cardknox-admin.js

```diff
@@ -14,57 +14,113 @@
  You should have received a copy of the GNU General Public License
  along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
-jQuery( function( $ ) {
-	'use strict';
+jQuery(function ($) {
+  "use strict";
 
-	/**
-	 * Object to handle Cardknox admin functions.
-	 */
-	var wc_cardknox_admin = {
+  /**
+   * Object to handle Cardknox admin functions.
+   */
+  var wc_cardknox_admin = {
+    getSecretKey: function () {
+      return $("#woocommerce_cardknox_transaction_key").val();
+    },
 
-		getSecretKey: function() {
-			return $( '#woocommerce_cardknox_transaction_key' ).val();
-		},
+    /**
+     * Initialize.
+     */
+    init: function () {
+      $("input#woocommerce_cardknox_capture")
+        .change(function () {
+          if ($(this).is(":checked")) {
+            $("#woocommerce_cardknox_auth_only_order_status")
+              .closest("tr")
+              .hide();
+          } else {
+            $("#woocommerce_cardknox_auth_only_order_status")
+              .closest("tr")
+              .show();
+          }
+        })
+        .change();
+    },
+  };
 
-		/**
-		 * Initialize.
-		 */
-		init: function() {
-
-			$( 'input#woocommerce_cardknox_capture' ).change(function() {
-				if ( $( this ).is( ':checked' ) ) {
-					$( '#woocommerce_cardknox_auth_only_order_status' ).closest( 'tr' ).hide();
-				} else {
-					$( '#woocommerce_cardknox_auth_only_order_status' ).closest( 'tr' ).show();
-				}
-			}).change();
-		}
-
-
-	};
-
-	wc_cardknox_admin.init();
+  wc_cardknox_admin.init();
 });
 
-jQuery(function(a){(function(){a("input#woocommerce_cardknox-applepay_applepay_capture").change(function(){a(this).is(":checked")?a("#woocommerce_cardknox-applepay_applepay_auth_only_order_status").closest("tr").hide():a("#woocommerce_cardknox-applepay_applepay_auth_only_order_status").closest("tr").show()}).change()})()});
+jQuery(function (a) {
```

Comment:
Where is this minified code from? Is there a way to get the un-minified version?

Reply:
![image](https://github.com/cardknox/woocommerce-gateway-cardknox/assets/49514170/bb65500a-f164-4642-b091-07820ac296f8)

**I have already added both version, could you please check.**

Reply:
I have replaced a with $ sign.